jobs/mongodb_migration/src/mongodb_migration/collector.py

```diff
@@ -330,4 +330,13 @@ def get_migrations(self) -> list[Migration]:
                 version="20240221160700",
                 description="merge 'split-first-rows-from-streaming' and 'split-first-rows-from-parquet' responses to 'split-first-rows'",
             ),
+            CacheDeletionMigration(
+                cache_kind="split-duckdb-index",
+                version="20240221134400",
+            ),
+            CacheRenamingMigration(
+                cache_kind="split-duckdb-index-010",
+                new_cache_kind="split-duckdb-index",
+                version="20240221134500",
+            ),
```

Comment:
note that we should also do the same with the jobs, if any, or the workers will break when they pick one.

Reply:
Thanks! I forgot it